From 2586adfd1fe30d49e4ff0ca849eb3bfa3e9279ca Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Thu, 23 May 2024 13:13:05 +0800
Subject: [PATCH 052/109] WIN2030-15279:feat(FreeRTOSConfig.h):Set
 configTOTAL_HEAP_SIZE to 32KB

Changelogs:
1.Set configTOTAL_HEAP_SIZE to 32KB

Change-Id: I8f6ec01a5ce300c6c2b7f869a1d035465f0843b9
---
 Core/Inc/FreeRTOSConfig.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Core/Inc/FreeRTOSConfig.h b/Core/Inc/FreeRTOSConfig.h
index 242ebe3..a1dee2d 100644
--- a/Core/Inc/FreeRTOSConfig.h
+++ b/Core/Inc/FreeRTOSConfig.h
@@ -68,7 +68,7 @@
 #define configTICK_RATE_HZ                       ((TickType_t)1000)
 #define configMAX_PRIORITIES                     ( 56 )
 #define configMINIMAL_STACK_SIZE                 ((uint16_t)128)
-#define configTOTAL_HEAP_SIZE                    ((size_t)(30 * 1024))
+#define configTOTAL_HEAP_SIZE                    ((size_t)(32 * 1024))
 #define configMAX_TASK_NAME_LEN                  ( 16 )
 #define configUSE_TRACE_FACILITY                 1
 #define configUSE_STATS_FORMATTING_FUNCTIONS     1
-- 
2.25.1

